uadk_tool/Makefile.am

```diff
@@ -2,8 +2,8 @@ ACLOCAL_AMFLAGS = -I m4 -I./include
 AUTOMAKE_OPTIONS = foreign subdir-objects
 AM_CFLAGS=-Wall -Werror -fno-strict-aliasing -I$(top_srcdir) -I$(top_srcdir)/benchmark/include \
 			-pthread
-AM_CFLAGS += -fPIC -fPIE -pie -fstack-protector-strong -D_FORTIFY_SOURCE=2 \
--O2 -ftrapv -Wl,-z,now -Wl,-s
```

Comment:
The configs is necessary. Why do we need to remove these?

Reply:
build with clang reports error
 ./cleanup.sh
    ./autogen.sh
    ./configure CC=clang
    
  clang-12: error: -Wl,-z,now: 'linker' input unused [-Werror,-Wunused-command-line-argument]
    clang-12: error: -Wl,-s: 'linker' input unused [-Werror,-Wunused-command-line-argument]
    clang-12: error: -Wl,-z,now: 'linker' input unused [-Werror,-Wunused-command-line-argument]
    clang-12: error: -Wl,-s: 'linker' input unused [-Werror,-Wunused-command-line-argument]
    clang-12clang-12: : errorerror: : -Wl,-z,now: 'linker' input unused [-Werror,-Wunused-command-line-argument]-Wl,-z,now: 'linker' input unused [-Werror,-Wunused-command-line-argument]       
    clang-12: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]

Reply:
These compiled items have requirements in the internal security specifications and cannot be modified.

Reply:
is clang build is another requirement from openEuler?
why there is a patch to fix clang build in src-openeuler
https://gitee.com/src-openeuler/libwd/blob/master/0001-support-clang-compile.patch